### ISSUE TITLE
6.03 | Shop size issues

### ISSUE THREAD

**Issue by AfterAllImSoul** [44madfire/Growing-Rivalry#49]
This is kindof a mix of 2 bugs, but if you shop for cards you can get yourself to negative size. Then if you visit the _*better*_ shop, she'll be kind enough to reset you to 0.

**Comment by AfterAllImSoul** [comment]
well now I'm confused. I visited her day 2 and she gave me another ~850 height. I'm not sure exactly what's going on, but that's approx the amount I was negative when she reset me. Once buy is clicked in Lilith's shop 178cm becomes 1013. I've attached a savefile, but Github wont upload rmmzsave files, so just change the extension back.

[file20.txt](https://github.com/44madfire/Growing-Rivalry/files/14961177/file20.txt)
